docs/FHIRSearchConfiguration.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: default
+title: IBM FHIR Server - Search Configuration Overview
+date:   2019-10-09 08:37:05 -0400
+permalink: /fhirsearchconfigurationoverview/
+markdown: kramdown
+---
+# IBM FHIR Server - Search Configuration Overview
+The [FHIR Specification](https://www.hl7.org/fhir/r4/search.html) defines a set of searchable fields for each resource type, and corresponding RESTful API.  The IBM FHIR Server supports searching using the specification defined search parameters and using tenant specific search parameters (extensions). 
+
+Additionally, the IBM FHIR Server supports searching on additional fields, including:
```

Comment:
This should maybe be "Specifically" or other transition since the previous sentence already mentions searching on tenant-specific search parameters.

Reply:
right, fixed up